src/assets/components/CreateThought/CreateThought.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import "./CreateThought.css"; 
+
+const APIURL = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts";
```

Comment:
you already defined the APIURL in the ThoughtFeed.jsx. You could have passed the APIURL to this component.